src/ILCompiler/src/RemovingILProvider.cs

```diff
@@ -150,5 +178,6 @@ public enum RemovedFeature
     {
         Etw = 0x1,
         FrameworkResources = 0x2,
+        Globalization = 0x4,
```

Comment:
Mention these in https://github.com/dotnet/corert/blob/master/Documentation/using-corert/optimizing-corert.md ?  Or are they too special to be mentioned there?

Reply:
I will - I didn't surface these from CSPROJ properly yet. I want to match how these are surfaced in the CSPROJ for IL Linker and didn't get around to finding where/how that's done.